### PR TITLE
refactor: [M3-6305] - MUI v5 Migration SRC > Components > ConfirmationDialog

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝

- Migrate ConfirmationDialog component from JSS to tss-react.
- Use named export instead of default

## How to test 🧪
- Validate places in the app where ConfirmationDialog is used.

**How do I run relevant unit or e2e tests?**

- yarn test
- yarn cy:run

### CODE REVIEW

File: packages/manager/src/features/linodes/PowerActionsDialogOrDrawer.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import * as React from 'react';
 import { compose } from 'recompose';
 import ActionsPanel from 'src/components/ActionsPanel';
 import Button from 'src/components/Button';
-import Dialog from 'src/components/ConfirmationDialog';
+import { ConfirmationDialog } from 'src/components/ConfirmationDialog/ConfirmationDialog';
```
Comment (jaalah-akamai) [inline packages/manager/src/features/linodes/PowerActionsDialogOrDrawer.tsx:12]: Named exports make this so much clearer, especially when we have an actual `Dialog` component!

**Review by jaalah-akamai — APPROVED** [review]

File: packages/manager/src/components/ConfirmationDialog/ConfirmationDialog.tsx [hunk 2]
```diff
@@ -35,8 +35,8 @@ export interface Props extends DialogProps {
   title: string;
 }
 
-const ConfirmationDialog: React.FC<Props> = (props) => {
-  const classes = useStyles();
+export const ConfirmationDialog = (props: Props) => {
```
Comment (mjac0bs) [inline packages/manager/src/components/ConfirmationDialog/ConfirmationDialog.tsx:38]: Can we rename the Props interface to be called `ConfirmationDialogProps` to help this component use our [best practices](https://github.com/linode/manager/blob/develop/docs/development-guide/02-component-structure.md)?

**Review by mjac0bs — APPROVED** [review]
Component in the UI and in Storybook looks good to me.

I'm approving pending a changelog update and (optionally, but would be nice to do) renaming the Props interface + updating the imports in the necessary files.

File: CHANGELOG.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 - MUI v5 Migration - `Features > Billing` #8933
 - MUI v5 Migration - `Components > CheckBox` #8980
 - MUI v5 Migration - `Components > BackupStatus` #8960
+- MUI v5 Migration - `SRC > Components > ConfirmationDialog` #9016
```
Comment (mjac0bs) [inline CHANGELOG.md:47]: We'll want this in the [Unreleased] section, not the last (Monday's) release.

**Pull request merged**